src/app/auth/login/login.component.ts

```diff
@@ -87,17 +90,23 @@ export class LoginComponent implements OnInit {
 	}
 
 	redirect() {
-		debugger;
+		let url;
 		this.cookieService.set('org', this.org.url, null, '/', '.newvote.org');
-		// window.location.href = this.org.authUrl;
 		console.log('cordova: ', window['cordova']);
+
+		if (this.org.authEntityId) {
+			url = this.adminLogin ? `${this.org.authUrl}` : `${this.org.authUrl}?entityID=${this.org.authEntityId}`;
+		} else {
+			url = `${this.org.authUrl}`;
+		}
+
 		if (window['cordova']) {
 			console.log('found cordova opening with inappbrowser');
 			const CORDOVA = window['cordova'];
-			CORDOVA.InAppBrowser.open(this.org.authUrl, '_blank');
-		} else {
-			window.open(this.org.authUrl, '_self');
+			return CORDOVA.InAppBrowser.open(url, '_blank');
 		}
+		
+		return window.open(url, '_self');
```

Comment:
this line here is all you need now, the if/else don't work

Reply:
Are you sure? Because url is a variable that's set in the if/else. Do you want me to change it back to just being ` window.open(this.org.authUrl, '_self');`?

Reply:
sorry I meant in the context of the previous comment of "you can remove all this cordova junk" but you got it right in the commit anyway